Extensions/TweenBehavior/tweenruntimebehavior.js

```diff
@@ -7,8 +7,8 @@
  */
 gdjs.TweenRuntimeBehavior = function (runtimeScene, behaviorData, owner) {
   gdjs.RuntimeBehavior.call(this, runtimeScene, behaviorData, owner);
-
   /** @type Object.<string, gdjs.TweenRuntimeBehavior.TweenInstance > */
+
```

Comment:
See if you can revert this file to the state that is in master :)

Reply:
If you struggle with this, let it alone and I'll revert the file when merging the PR.

Reply:
I guess I'm indeed struggling. ^^